pkgs/top-level/all-packages.nix

```diff
@@ -15945,6 +15947,15 @@ in
 
   blas = callPackage ../development/libraries/science/math/blas { };
 
+  clblas-cuda = callPackage ../development/libraries/science/math/clblas/cuda {
+    cudatoolkit = pkgs.cudatoolkit75;
+    inherit (linuxPackages) nvidia_x11;
+    python = pkgs.python34Packages.python;
```

Comment:
`clblas-cuda` gets built fine with other versions of of `pythonPackages` and `boost`. I have checked the four combinations corresponding to `python-2.7`, `python-3.5` and `boost-1.59`, `boost-1.60`. I will adjust the definition of `clblas-cuda` so that it respects the functoriality with respect to morphisms of `python` and `boost`. On the other hand, it is currently necessary to keep the hard-wired version of `cudatoolkit`: `cudatoolkit-7.0` corresponds to OpenCl 1.1, and `cudatoolkit-7.5` corresponds to OpenCL 1.2, while `clblas` wants to have at least version 1.2. I will push the updates after the `Theano` tests, and I have to check if one really needs `python-3.4` for that.